src/__tests__/standalone-needlessDisables-test.js

```diff
@@ -0,0 +1,61 @@
+import path from "path"
```

Comment:
@davidtheclark `needDisables-test.js` tests only `needDisables.js`, but `standalone-needlessDisables-test.js` used `standalone`, we can change some line in this file and got errors, be good add tests on this case.